aws/resource_aws_elasticache_cluster_test.go

```diff
@@ -514,15 +514,16 @@ resource "aws_elasticache_cluster" "bar" {
     cluster_id = "tf-%s"
     engine = "memcached"
     node_type = "cache.m1.small"
-    num_cache_nodes = 2
+    num_cache_nodes = 3
     port = 11211
     subnet_group_name = "${aws_elasticache_subnet_group.bar.name}"
     security_group_ids = ["${aws_security_group.bar.id}"]
     parameter_group_name = "default.memcached1.4"
     az_mode = "cross-az"
     availability_zones = [
         "us-west-2a",
-        "us-west-2b"
+        "us-west-2b",
```

Comment:
The order does not matter.  Does Terraform have a type where it sorts the list first before comparing for changes (for example, if this list changes to "2a 2b 2a" from "2a 2a 2b", that's not actually something that needs to be updated.

Reply:
@cep21 Sorting list is good idea to avoid not necessary update. Thank you!